Makefile

```diff
@@ -1,4 +1,8 @@
 include app.env
+export $(shell sed 's/=.*//' app.env)
```

Comment:
Do we? I can't say that I know exactly what this does because I did just copy it from StackOverflow but my assumption is that it exports the env variables in app.env?

This is necessary since the app is started on the host when you run `make up-dev`

Reply:
If you look at [this link](https://sed.js.org/?gist=f49803efc6b5e55b0ccad743e8798c5e)
You will see that all the values are stripped from the env

It shouldn't matter since they are provided in other places I think

Reply:
When you're running `make up-dev` this is the line that is meant to initialize the env vars, they don't get initialized anywhere else.

For whatever reason if comment out this line on my local setup it says that the env vars are not defined 🤷‍♂